docs/sctp/sctp.md

```diff
@@ -0,0 +1,2 @@
+# SCTP Golang Library 
+
```

Comment:
you should give a reference to the SCTP documentation, and some notes about how you chose to implement it

Reply:
Yeah I will add more to it.